airflow/api_connexion/endpoints/user_endpoint.py

```diff
@@ -49,7 +49,7 @@ def get_users(limit, order_by='id', offset=None):
     to_replace = {"user_id": "id"}
     allowed_filter_attrs = [
         "user_id",
-        'id',
+        "id",
```

Comment:
```suggestion
        'id',
```

Otherwise unchanged file  -- let's keep our git history clean.

Reply:
done